server/controllers/requestController.js

```diff
@@ -0,0 +1,44 @@
+import RequestHelper from '../helpers/requestHelper';
+import users from '../helpers/authHelpers';
+
+/**
+ * This class contains all methods
+ * required to approve/reject
+ *  atrip requests.
+ */
+class RequestController {
+  /**
+   * This method approve the trip request.
+   * @param {object} req The user's trip request.
```

Comment:
@Nigorjeanluc  are you sure req stands for trip request? or http request? Think twice. Thank you